### DISCUSSION TITLE
Discontinuation of Insanity Remastered.

### DISCUSSION THREAD

**Discussion by BudgetAirpods** [BudgetAirpods/InsanityRemastered#40] [Announcements]
Hello everyone!

Yes, the title is true. I'm discontinuing Insanity Remastered. The reasoning behind this is simple: I want to move onto other projects.

This started out as a fun little project, with some of the hallucinations coming from another game idea I had just scrapped not too long ago. I did not think this many people would download it and actually get scared by it.

I will try my hardest to fix the bugs caused by this update, but after that there will be no more updates. (Including compatibility issues).

The mod will be left up and available for download, however as stated before, I will not be updating it anymore.

If you want to make your own port/version of this mod, you are free to do so and can ask questions about the code anytime. (Yes, I'm aware my code isn't exactly the prettiest).

If anyone has some minor QoL or suggestions for the mod, now will be the chance to have them added.

**Comment by HossBigft** [comment]
sad news, I like this mod. There will be no way to turn off auditory hallucinations, I guess?

**Reply by BudgetAirpods** [reply]
There is an option but from the looks of it, it doesn't work. I'm going to fix this in the last update. 

The last update will be fixing bugs caused by the latest update.

**Reply by Hodurlein** [reply]
Well, can you at least give options in the configuration to tune down auditory/visual hallucinations by percentage and be able to turn off slowness caused by panic attacks, etc.? I really loved this mod. It was a unique experience. The last update just broke it, but nonetheless, it was truly scary and perfect; the game really needed an insanity mod like this perfect idea, but it was sad to see it go away. wishing you the best on your next projects!

**Reply by BudgetAirpods** [reply]
Yeah. I'll work on adding those.

I really do like the idea of this mod and was surprised when I couldn't find anything similar to it. I actually almost ended it prematurely after the updated that added Skinwalker integration due to lack of motivation.

I originally had the idea for a game based around an insanity mechanic. When I found out Lethal Company had one, I decided to transfer most of my ideas over to it.

Like I said though, I want to work on other projects. The main thing I'm working on now is creating an AI similar to that of the Alien from Alien: Isolation so I could potentially use it in a future game.

**Reply by anon-cfg** [reply]
Very sad to see you go, love this mod. Hope you enjoy your next project, sounds like it could be really cool.

**Comment by TheXeleX** [comment]
Hi, I love ur mod but I'm having an issue cause you said that it's a client-side mod so when i have it and my firend don't, it sould work right? And currentrly it dosen't so I don't know what to do.

If u have any idea of what I should do, let me know ^^

**Reply by BudgetAirpods** [reply]
No. Your friends do not need this in order for it to work.

What do you mean by "doesn't work"? Nothing is happening in game?

**Reply by TheXeleX** [reply]
Yeah, no allucinations at all. It's like the mod is not intalled. But when everyone had it at the begining, it was working and then one of my friend decided to disable it and now it's not working for all

**Reply by BudgetAirpods** [reply]
Are you loading it through thunderstore? I think the mod tends to break just because the sounds don't load properly

**Reply by TheXeleX** [reply]
Yeah, I'm using thunderstore to launch it

**Comment by Mintybutt** [comment]
It's a shame, this is one of the best mods out there at the moment. I wish you luck on your other projects.

And, I can probably surmise from "I will not be updating it anymore," but just to be perfectly clear... You're not even going to update for newer versions of LC, like if the mod happens to break entirely? Or does this fall under 'compatibility issues'?

**Reply by BudgetAirpods** [reply]
I will try my best to keep it up to date with newer versions of Lethal Company; however, I cannot guarantee that will always be true.

And yes, any compatibility issues will not be fixed. (EDIT: Depending on the severity of them, that can change. If another mod breaks IR or vice versa, I will try to fix it.)

**Comment by HamedAlmanthari** [comment]
Oh damn i just saw this. I understand where you’re coming from but there is a very major issue with the latest version that was supposed to be fixed with the update but it appears that you uploaded wrong files or something to thunderstore. I’ll leave the link here. https://github.com/BudgetAirpods/InsanityRemastered/issues/34#issuecomment-1912598766

**Comment by justinfn06** [comment]
Im sad to hear that. Just a question, but what exactly does the "experimental version of skinwalker" do? is this not implemented at all, or is just a partially changed skinwalker?

**Comment by Ink-01** [comment]
will you ever consider releasing the source code?
Edit: nvm, just saw that the branches has it
